### PR TITLE
chore!: CDK 1.94, naming, eks 1.19 

### PR DESCRIPTION
**Pull request by Miradorn**
* Updates cdk to 1.94.1
* Unifies naming of resources and fixes scopes of construct resources
* update to eks 1.19

BREAKING CHANGE: Due to naming and scope problem, it's not possible to
upgrade existing super-eks clusters without running into problems.
If you can, it's probably easiest to just create a new stack and
migrate your workloads.
A normal update will try to create a new cluster and delete the old one
which will lead to downtime.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -13,9 +13,17 @@ Name|Description
 Name|Description
 ----|-----------
 [AddonProps](#superluminar-io-super-eks-addonprops)|Specific properties for EKS managed add-ons.
+[NodeTaint](#superluminar-io-super-eks-nodetaint)|*No description*
```
Comment (bracki) [inline API.md:16]: No description sounds suspicious.

File: API.md [hunk 2]
```diff
@@ -13,9 +13,17 @@ Name|Description
...
 [SuperEksProps](#superluminar-io-super-eks-supereksprops)|Constructor properties for SuperEks.
 
 
+**Enums**
+
+Name|Description
+----|-----------
+[TaintEffect](#superluminar-io-super-eks-tainteffect)|*No description*
```
Comment (bracki) [inline API.md:24]: Same as above.

File: API.md (hunk 1 above)
Comment (bracki) [reply API.md:16]: No description sounds suspicious.

**Review by bracki — CHANGES_REQUESTED** [review]
Apart from the comments does this have any impact on potential users? Or is it an internal thing?

**Comment by Miradorn** [comment]
> Apart from the comments does this have any impact on potential users? Or is it an internal thing?

TBH I'm not totally sure, and I'm going to test it.

Right now I'd guess that the ASG backing the NodeGroup would be updated and new nodes would be rotated in because the userdata does change.

I'd argue that that is OK, since Node rotation is a "normal" thing that happens all the time.

But I will test it before merging.

**Comment by Miradorn** [comment]
Update: unfortunately we have to introduce a breaking change, since the cluster cannot be migrated due to naming and scope conflicts.
Since we're pre 1.0.0 we're ok with this.

**Review by bracki — APPROVED** [review]

**Pull request merged**